src/main/java/org/jitsi/jicofo/bridge/RegionBasedBridgeSelectionStrategy.java

```diff
@@ -15,68 +27,4 @@
     RegionBasedBridgeSelectionStrategy()
```

Comment:
This is now a no-op? Why not remove that class if that's the case?

Reply:
I'm not sure we want this as a no-op; now the scheme is implemented in the base class and so it's inherited by the specialised classes, which may not be a good idea